src/components/annotations/attributes.js

```diff
@@ -131,14 +131,31 @@ module.exports = {
         role: 'style',
         description: 'Sets the width (in px) of annotation arrow.'
     },
+    absolutetail: {
+        valType: 'boolean',
+        dflt: false,
+        role: 'style',
+        description: [
+            'Indicates if the tail of this arrow is a point in ',
```

Comment:
Wouldn't it be better, more customizable and more _plotly.js-esque_ to have this functionality be set with a pair of attributes, one for each axis?

I'm thinking about: 

```js
{
  axref: 'x',  // or 'pixel' (or 'paper' down the road) similar to 'xref' but for the arrow tail
  ayref: 'y'  
}
```

@mdfederici But this solution might have too many degrees of freedom for most cases. Can you think of a use case where a user would want the `x` arrow coordinates in pixel and the `y` in data coordinates (or vice-versa)?